content/general/authentication.textile

```diff
@@ -139,22 +139,91 @@ ably.requestToken({ clientId: 'client@example.com' }, function(err, token) {
      } */
 });
 
-h3(#capabilities-explained). Capabilities and Token Security explained
+h4(#ably-jwt-process). Ably JWT is created by your servers and passed to clients
 
-"API keys":https://support.ably.io/solution/articles/3000030054-what-is-an-app-api-key, like "tokens":#tokens, have a set of capabilities assigned to them that specify which "operations":#capability-operations (such as subscribe or publish) can be performed on which channels. However, unlike tokens, API keys are long-lived, secret and typically not shared with un-trusted clients.
+It is possible to use a JWT as a form of authentication with Ably, so long as it is formatted correctly, in what will be referred to as an *"Ably JWT":#ably-jwt*. It is possible within the Ably JWT to specify a lifetime and a set of capabilities, signing it with the relevant API key. 
 
-API keys and their capabilities are "configured using the dashboard":https://support.ably.io/support/solutions/articles/3000030502-setting-up-and-managing-api-keys, they cannot be added or removed programmatically. Tokens on the other hand are designed to be shared with un-trusted clients, are short-lived, and significantly, they are configured and issued programmatically using the Ably client libraries or directly from the "Ably REST API":/rest-api. See "selecting an authentication scheme":#selecting-auth to understand why token authentication, in most cases, is the preferred authentication scheme.
+This is similar to signing an "Ably TokenRequest":/realtime/authentication#request-token, but the client does not need to then request an Ably Token, instead being able to use the Ably JWT as a token in itself. Any third-party library may be used to create the JWT without a connection ever needing to be established with Ably. This can be useful for situations where an Ably client is not available, such as an embedded MQTT client.
```

Comment:
".. to then request ..." -> " ... then to request ... ".

Any compliant third-party JWT library may be used to create the JWT without requiring the token to be issued by Ably. This can be useful for situations where an Ably client library is not available, such as an embedded device connecting to Ably via MQTT.